<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HgRevisionsSelectionDialog</class>
 <widget class="QDialog" name="HgRevisionsSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mercurial Diff</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="rev1GroupBox">
     <property name="title">
      <string>Revision &amp;1</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="number1Button">
        <property name="toolTip">
         <string>Select to specify a revision by number</string>
        </property>
        <property name="text">
         <string>Number</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="number1SpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a revision number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight</set>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="id1Button">
        <property name="toolTip">
         <string>Select to specify a revision by changeset id</string>
        </property>
        <property name="text">
         <string>Id:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="id1Edit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a changeset id</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="tag1Button">
        <property name="toolTip">
         <string>Select to specify a revision by a tag</string>
        </property>
        <property name="text">
         <string>Tag:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="tag1Combo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a tag name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="branch1Button">
        <property name="toolTip">
         <string>Select to specify a revision by a branch</string>
        </property>
        <property name="text">
         <string>Branch:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="branch1Combo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a branch name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="bookmark1Button">
        <property name="toolTip">
         <string>Select to specify a revision by a bookmark</string>
        </property>
        <property name="text">
         <string>Bookmark:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QComboBox" name="bookmark1Combo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter a bookmark name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QRadioButton" name="tip1Button">
        <property name="toolTip">
         <string>Select tip revision of repository</string>
        </property>
        <property name="text">
         <string>TIP</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QRadioButton" name="prev1Button">
        <property name="toolTip">
         <string>Select revision before last commit</string>
        </property>
        <property name="text">
         <string>PREV</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QRadioButton" name="none1Button">
        <property name="text">
         <string>No revision selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="rev2GroupBox">
     <property name="title">
      <string>Revision &amp;2</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="number2Button">
        <property name="toolTip">
         <string>Select to specify a revision by number</string>
        </property>
        <property name="text">
         <string>Number</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="number2SpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a revision number</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight</set>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="id2Button">
        <property name="toolTip">
         <string>Select to specify a revision by changeset id</string>
        </property>
        <property name="text">
         <string>Id:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="id2Edit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a changeset id</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="tag2Button">
        <property name="toolTip">
         <string>Select to specify a revision by a tag</string>
        </property>
        <property name="text">
         <string>Tag:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="tag2Combo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a tag name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="branch2Button">
        <property name="toolTip">
         <string>Select to specify a revision by a branch</string>
        </property>
        <property name="text">
         <string>Branch:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="branch2Combo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Enter a branch name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="bookmark2Button">
        <property name="toolTip">
         <string>Select to specify a revision by a bookmark</string>
        </property>
        <property name="text">
         <string>Bookmark:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QComboBox" name="bookmark2Combo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter a bookmark name</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QRadioButton" name="tip2Button">
        <property name="toolTip">
         <string>Select tip revision of repository</string>
        </property>
        <property name="text">
         <string>TIP</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QRadioButton" name="prev2Button">
        <property name="toolTip">
         <string>Select revision before last commit</string>
        </property>
        <property name="text">
         <string>PREV</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="3">
       <widget class="QRadioButton" name="none2Button">
        <property name="text">
         <string>No revision selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>number1Button</tabstop>
  <tabstop>number1SpinBox</tabstop>
  <tabstop>id1Button</tabstop>
  <tabstop>id1Edit</tabstop>
  <tabstop>tag1Button</tabstop>
  <tabstop>tag1Combo</tabstop>
  <tabstop>branch1Button</tabstop>
  <tabstop>branch1Combo</tabstop>
  <tabstop>bookmark1Button</tabstop>
  <tabstop>bookmark1Combo</tabstop>
  <tabstop>tip1Button</tabstop>
  <tabstop>prev1Button</tabstop>
  <tabstop>none1Button</tabstop>
  <tabstop>number2Button</tabstop>
  <tabstop>number2SpinBox</tabstop>
  <tabstop>id2Button</tabstop>
  <tabstop>id2Edit</tabstop>
  <tabstop>tag2Button</tabstop>
  <tabstop>tag2Combo</tabstop>
  <tabstop>branch2Button</tabstop>
  <tabstop>branch2Combo</tabstop>
  <tabstop>bookmark2Button</tabstop>
  <tabstop>bookmark2Combo</tabstop>
  <tabstop>tip2Button</tabstop>
  <tabstop>prev2Button</tabstop>
  <tabstop>none2Button</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HgRevisionsSelectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HgRevisionsSelectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>number1Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>number1SpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>number2Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>number2SpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>id1Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>id1Edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>id2Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>id2Edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tag1Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>tag1Combo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>branch1Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>branch1Combo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tag2Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>tag2Combo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>branch2Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>branch2Combo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookmark1Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>bookmark1Combo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookmark2Button</sender>
   <signal>toggled(bool)</signal>
   <receiver>bookmark2Combo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
